from builtins import Exception


class InstantiateCSVError(Exception):
    """
    Создание экземпляра класса нового исключение InstantiateCSVError.

    """
    def __init__(self, *args, **kwargs):  # real signature unknown
        pass
